unit_test/fuzzing/test_responder/test_spdm_responder_event_types/event_types.c

```diff
@@ -0,0 +1,104 @@
+{
+    libspdm_test_context_t *spdm_test_context;
+    libspdm_context_t *spdm_context;
+    libspdm_session_info_t *session_info;
+    uint32_t session_id;
+    size_t response_size;
+    uint8_t response[LIBSPDM_MAX_SPDM_MSG_SIZE];
+
+    spdm_test_context = *State;
+    spdm_context = spdm_test_context->spdm_context;
+
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_13 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+    spdm_context->connection_info.connection_state = LIBSPDM_CONNECTION_STATE_NEGOTIATED;
+    spdm_context->response_state = LIBSPDM_RESPONSE_STATE_NORMAL;
+
+    spdm_context->connection_info.capability.flags =
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_EVENT_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_ENCRYPT_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MAC_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_KEY_EX_CAP;
+
+    spdm_context->local_context.capability.flags =
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_ENCRYPT_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MAC_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_KEY_EX_CAP |
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_EVENT_CAP;
+
+    spdm_context->connection_info.algorithm.base_hash_algo = m_libspdm_use_hash_algo;
+    spdm_context->connection_info.algorithm.base_asym_algo = m_libspdm_use_asym_algo;
+    spdm_context->connection_info.algorithm.dhe_named_group = m_libspdm_use_dhe_algo;
+    spdm_context->connection_info.algorithm.aead_cipher_suite = m_libspdm_use_aead_algo;
+
+    session_id = 0xFFFFFFFF;
+    spdm_context->latest_session_id = session_id;
+    spdm_context->last_spdm_request_session_id_valid = true;
+    spdm_context->last_spdm_request_session_id = session_id;
+    session_info = &spdm_context->session_info[0];
+    libspdm_session_info_init(spdm_context, session_info, session_id, true);
+    libspdm_secured_message_set_session_state(
+        session_info->secured_message_context,
+        LIBSPDM_SESSION_STATE_ESTABLISHED);
+
+    response_size = sizeof(response);
+    libspdm_get_response_supported_event_types(spdm_context,
+                                               spdm_test_context->test_buffer_size,
+                                               spdm_test_context->test_buffer,
+                                               &response_size, response);
+}
+
+libspdm_test_context_t m_libspdm_responder_event_types_test_context = {
+    LIBSPDM_TEST_CONTEXT_VERSION,
+    false,
+};
+
+void libspdm_run_test_harness(void *test_buffer, size_t test_buffer_size)
+{
+    void *State;
+    spdm_message_header_t *spdm_request_header;
+    libspdm_setup_test_context(&m_libspdm_responder_event_types_test_context);
+
+    spdm_request_header = (spdm_message_header_t*)test_buffer;
+
+    if (spdm_request_header->request_response_code != SPDM_SUPPORTED_EVENT_TYPES) {
+        spdm_request_header->request_response_code = SPDM_SUPPORTED_EVENT_TYPES;
```

Comment:
Here `SPDM_SUPPORTED_EVENT_TYPES` should be changed to `SPDM_GET_SUPPORTED_EVENT_TYPES`, right?